Dockerfile.bench

```diff
@@ -25,27 +25,19 @@ RUN set -x \
  && apt-get update \
  && apt-get install --no-install-recommends -y build-essential python3-dev python3-pip \
  && rm -rf /var/lib/apt/lists/*
-# https://stackoverflow.com/questions/52904639/dockerfile-with-pip-grpcio-is-very-slow-to-build => fix not applied yet
 
 RUN set -x \
   && apt-get update \
-  && apt-get install -y openjdk-11-jdk-headless \
-  && rm -rf /var/lib/apt/lists/*
-
-RUN set -x \
-  && apt-get update \
-  && apt-get install -y apt-file \
-  && apt-file update \
-  && apt-get install -y vim \
-  && apt-get install -y wget
+  && apt-get install -y apt-file vim wget
```

Comment:
`rm -rf /var/lib/apt/lists/*`... or maybe if the savings aren't significant (check `docker history <image>`), just remove this `rm` command from other layers as well

Reply:
Also... I don't think `apt-file` is will be particularly useful in the dev image (?)

Reply:
ok, it's not much. 
sure, I just took some of that from your original docker images.